modules/olm-installing-operators-from-operatorhub-configure.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * operators/user/olm-installing-operators-in-namespace.adoc
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+//
+// Module watched for changes by Ecosystem Catalog team:
+// https://projects.engineering.redhat.com/projects/RHEC/summary
+
+
+[id="olm-installing-operators-from-operatorhub-configure_{context}"]
+= Configuring {product-title} to use Red Hat Operators
+
+In {product-title}, Red Hat Operators are not available by default. You can access and install these Operators if you have a pull secret from the Red Hat OpenShift Cluster Manager site by editing the Operator Hub custom resource (CR). 
```

Comment:
My brain is low on power, so I'm not sure but should this be "the `OperatorHub` custom resource (CR)"?
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references